_uk/scala3/guides.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: inner-page-parent
+title: Посібники по Scala 3
+language: uk
+scala3: true
+
+guides:
```

Comment:
this page should not exist (i.e. delete it and any links to it) because all the links should have been moved to `/uk/overviews/index.html` already

Reply:
remember to update `_data/overviews-uk.yml` with the new translations

Reply:
so actually this data does not need to be updated because the page generator automatically substitutes the translated url if available

Reply:
Thank you